rotkehlchen/db/history_events.py

```diff
@@ -31,9 +32,22 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
-HISTORY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier, sequence_index,
-timestamp, location, location_label, asset, amount, usd_value, notes,
-type, subtype, counterparty, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+HISTORY_BASE_ENTRY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier,
+ sequence_index, timestamp, location, location_label, asset, amount, usd_value, notes,
+ type, subtype) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+
+EVM_EVENT_INSERT = """INSERT OR IGNORE INTO evm_events_info(identifier, protocol, product,
+address, extra_data) VALUES (?, ?, ?, ?, ?);"""
+
+HISTORY_BASE_ENTRY_FIELDS = """history_events.identifier, history_events.event_identifier,
+ history_events.sequence_index, history_events.timestamp, history_events.location,
+ history_events.location_label, history_events.asset, history_events.amount,
+ history_events.usd_value, history_events.notes, history_events.type, history_events.subtype"""
```

Comment:
I added the table name here just to be more explicit. In case we will add a column with one of these names to another table later. To don't get ambiguous columns.

But perhaps for now the table name can be removed.

-----

As for the small tab (it is a single whitespace) -- it is to have a uniform format of the query. After commas we have whitespaces, but when there is a new line the whitespace needs to be at the beginning of the new line (can't be at the end of the previous line because then linters complain due to a trailing whitespace).